Header header
geometry_msgs/Pose pose
geometry_msgs/Vector3 translation_corrections
float64 translation_correction
geometry_msgs/Vector3 rotation_correction_axis
float64 rotation_correction_angle
float64 outlier_percentage
float64 root_mean_square_error_inliers
float64 root_mean_square_error_of_last_registration_correspondences
uint64 number_inliers
uint64 number_points_registered
float64 inliers_angular_distribution
float64 outliers_angular_distribution
int64 number_of_registration_iterations_for_all_matchers
string last_matcher_convergence_state
geometry_msgs/PoseArray initial_pose_estimation_poses
